gnome-extra/cinnamon-settings-daemon/cinnamon-settings-daemon-6.4.3.ebuild

```diff
@@ -0,0 +1,118 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit meson flag-o-matic gnome2-utils python-any-r1 xdg
+
+DESCRIPTION="Cinnamon's settings daemon"
+HOMEPAGE="https://projects.linuxmint.com/cinnamon/ https://github.com/linuxmint/cinnamon-settings-daemon"
+SRC_URI="https://github.com/linuxmint/cinnamon-settings-daemon/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ LGPL-2+ LGPL-2.1 LGPL-2.1+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
+IUSE="+colord cups input_devices_wacom smartcard systemd wayland"
+
+RDEPEND="
+	>=dev-libs/glib-2.40.0:2[dbus]
+	dev-libs/libgudev
+	>=gnome-base/libgnomekbd-3.6:=
+	>=gnome-extra/cinnamon-desktop-6.4:0=
+	media-libs/fontconfig
+	>=media-libs/lcms-2.2:2
+	|| (
+		media-libs/libcanberra-gtk3
+		media-libs/libcanberra[gtk3(-),pulseaudio]
+	)
+	>=media-libs/libpulse-0.9.16[glib]
+	>=sys-auth/polkit-0.97
+	sys-libs/timezone-data:=
```

Comment:
the intention was for changes in `timezone-data` to trigger rebuilds of `cinnamon-settings-daemon`, since it generates a header from the zoneinfo data.

Unfortunately, I just now noticed that `timezone-data` doesn't use (sub-)slot operators, so this won't do anything. Are there any other ways to trigger rebuilds on dependency changes? Or should I file a bug to have `timezone-data` add sub-slots?